README.md

```diff
@@ -13,6 +13,24 @@ The driver is compatible across the entire line of UR robots -- from 3 kg payloa
 Check also [presentations and videos](ur_robot_driver/doc/resources/README.md) about this driver.
 
 
+## Installation
+
+The driver is available as several released packages. On Ubuntu, they can be installed with `apt` by running:
+
+```console
+$ sudo apt-get install ros-${ROS_DISTRO}-ur ros-${ROS_DISTRO}-ur-bringup
+```
+
+The driver can also be installed with `rosdep` by added the following tags to a package manifest in a developer's workspace:
+```yaml
+# package.xml
+<depend>ur</depend>
+<depend>ur_bringup</depend>
+```
+
+Lastly, run `rosdep install --from-paths src` to complete the installation.
```

Comment:
I am not sure whether it would make sense to add this to the README. First of all, the same comment for `ur_bringup` as above is valid.

Second, depending on the actual package it might not make sense to depend on the metapackage, but rather on the package that the downstream package is actually using (`ur_robot_driver`, `ur_description`, `ur_moveit_config`...)

And third, I am afraid that this will confuse new users more than it helps them. Developers familiar with this concept will be able to infer that possibility once they know that it is available as apt package. At least that would be my assumption.